src/butil/iobuf.h

```diff
@@ -246,7 +247,18 @@ friend class IOBufCutter;
     // Append the user-data to back side WITHOUT copying.
     // The user-data can be split and shared by smaller IOBufs and will be
     // deleted using the deleter func when no IOBuf references it anymore.
-    int append_user_data(void* data, size_t size, void (*deleter)(void*));
+    inline int append_user_data(void* data, size_t size, void (*deleter)(void*)) {
```

Comment:
This method should be defined in `iobuf_inl.h`

Reply:
OK